app/views/user/contribute/_finished_tracks.erb

```diff
@@ -0,0 +1,3 @@
+<li>
```

Comment:
Totally agree with you on this one this partial is not really need it, is the habit of moving things to partials. I will remove it